changelog/2.0.0.md

```diff
@@ -0,0 +1,140 @@
+- the module `semanticdb` no longer depends on `parsers` or other Scala-specific
+  language features. This means it is possible to implement Semantic DB
+  integrations for other languages than Scala.
+- scalahost has been renamed to `semanticdb-scalac`
+- `-P:semanticdb:failures:ignore/info/warning/error` to configure at which
+  level semanticdb errors are reported.
+- sbt-scalahost has been removed, add the semanticdb-scalac compiler plugin directly instead
+- New Denotation.isJavaDefined to determine if a symbol is compiled with javac.
+- Denotations are now emitted for all used symbols in a compilation unit.
+- Several naming resolution bugs have been fixed, for example [#960][]
+- Several Windows-related bugs have been fixed, our CI now runs tests on Appveyor for every PR
+  to prevent further regressions.
+
+```scala
+// before in project/plugins.sbt
+addSbtPlugin("org.scalameta" % "sbt-scalahost" % "1.8.0")
+// after in build.sbt
+addCompilerPlugin("org.scalameta" % "semanticdb-scalac" % "0.2.0" cross CrossVersion.full)
+scalaOptions += "-Yrangepos"
+scalaOptions += "-P:semanticdb:sourceroot:/path"
+```
+
+## Tokenizers
+
+- `enum` is now considered a keyword in the Dotty dialect
+- Several XML literal issues have been fixed, by @allanrenucci
+
+## Parsers
+
+- New JavaScript facade to parse strings to JSON object, install on npm
+  https://www.npmjs.com/package/scalameta-parsers, by @gabro
+- Interactively explore `scala.meta.Tree` on https://astexplorer.net/, by @gabro
+- Several parser bugs have been fixed, see [#942][], by @usrinivasan
+- Reject case classes without parameter lists for Scala 2.11+, [#929][], by @jonas
+
+## Trees
+
+- The Tree AST has significantly changed in order to
+  - TODO(xeno_by) motivate
+  - reduce complexity
+  - align closer with the tree API in [scalamacros/scalamacros](https://github.com/scalamacros/scalamacros).
+- New `Tree.productFields: Seq[String]` field to get the field names of a tree node, useful in combination
+  with `productIterator` and `productPrefix`.
+- New `Type.ImplicitFunction` tree node for implicit function types in Dotty dialect.
+
+## Quasiquotes
+
+- TODO(xeno_by): summarize bug fixes in https://github.com/scalameta/scalameta/pull/1014
+
+## IO
+
+- `scala.meta.AbsolutePath/RelativePath` are now thin wrappers around `java.nio.file.Path`.
+- `java.io.File` and `java.nio.file.Path` are now supported on Scala.js.
+- Node.js APIs are lazily loaded preventing errors in the browser, see [#1059][], by @jonas
+
+## Breaking changes
+
+| 1.8.0                                             | 2.0.0                                               |
+| -----                                             | -----                                               |
+| scala.collection.immmutable.Seq[T]                | List[T]                                             |
+| scala.meta.internal.ast                           | scala.meta.internal.trees                           |
```

Comment:
Even if we don't promise stability of internal packages, I think it's nice to include it here just in case some unfortunate soul like myself in the future is porting a library from 1.8 to 2.0.